File: Chitunion/Web(java)/src/main/java/com/xyauto/domain/TitleCarMapping.java
package com.xyauto.domain;

import java.util.Date;

import lombok.Data;

@Data
public class TitleCarMapping {
	private Integer recId;
	private Integer titleId;
	private Integer carBrandId;
	private Integer CSID;
	private Integer status;
	private Date createTime;
	private Integer createUserId;
}

File: Chitunion/Web(code refactoring)/XYAuto.ChiTu2018/XYAuto.ChiTu2018.Entities/Chitunion2017/Title_Car_Mapping.cs
namespace XYAuto.ChiTu2018.Entities.Chitunion2017
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    public partial class Title_Car_Mapping
    {
        [Key]
        public int RecID { get; set; }

        public int TitleID { get; set; }

        public int CarBrandID { get; set; }

        public int? CSID { get; set; }

        public int? Status { get; set; }

        public DateTime? CreateTime { get; set; }

        public int? CreateUserID { get; set; }
    }
}
